app/src/main/java/com/materiapps/gloom/rest/dto/repo/Repository.kt

```diff
@@ -0,0 +1,105 @@
+package com.materiapps.gloom.rest.dto.repo
+
+import com.materiapps.gloom.rest.dto.license.License
+import com.materiapps.gloom.rest.dto.user.User
+import kotlinx.datetime.LocalDateTime
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Repository(
+
+    val id: Long,
+
+    val name: String,
+
+    @SerialName("full_name")
```

Comment:
I usually specify SerialName for all fields but that's just a personal thing. Also please get rid of newlines